### PR TITLE
[0.7.1] Update documentation

### PR DESCRIPTION
**Pull request by mthrok**
* Bump version to 0.7.1
* Checkout changes from the latest

### CODE REVIEW

**Review by vincentqb — APPROVED** [review]
LGTM.

File: docs/source/functional.rst [hunk 1]
```diff
@@ -8,145 +8,176 @@ torchaudio.functional
 
 Functions to perform common audio operations.
 
-:hidden:`spectrogram`
```
Comment (vincentqb) [inline docs/source/functional.rst:11]: the functionals aren't moved in 0.7, right?

File: docs/source/functional.rst [hunk 2]
```diff
@@ -8,145 +8,176 @@ torchaudio.functional
...
-~~~~~~~~~~~~~~~~~~~~~
+:hidden:`Filtering`
```
Comment (vincentqb) [inline docs/source/functional.rst:11]: the functionals aren't moved in 0.7 but this subsection isn't visible, right?

Comment (mthrok) [reply docs/source/functional.rst:11]: It is visible. `hidden` here is a command to tell doxygen which class it should use to render this section.
Filtering subsection is for the improved user experience and has nothing to do with where the location these functions are defined.

Comment (vincentqb) [reply docs/source/functional.rst:11]: Right, this seems reasonable

**Pull request merged**